src/families/algorand/js-estimateMaxSpendable.ts

```diff
@@ -30,6 +31,8 @@ export const estimateMaxSpendable = async ({
     useAllAmount: true,
   };
 
+  tx.fees = await getEstimatedFees(mainAccount, tx);
```

Comment:
Maybe works, but fees are supposed to be set before already, in `prepareTransaction` [here](https://github.com/LedgerHQ/ledger-live-common/blob/5b012ceeca27be920ca31f3227b2b7bdcf51882c/src/families/algorand/js-prepareTransaction.ts#L39).

Would be good to understand how we get here without tx.fees :thinking:

Reply:
Maybe because https://github.com/LedgerHQ/ledger-live-common/blob/5b012ceeca27be920ca31f3227b2b7bdcf51882c/src/families/algorand/js-prepareTransaction.ts#L27 ?

Reply:
Bingo :slightly_smiling_face: good catch.
So I agree the simplest is to call `getEstimatedFees` fees twice indeed...

Reply:
fix in 5e7cc0d9a262e504b2371bd91526c7548adb1b2e